### ISSUE TITLE
`bw2calc` does not recognize `scikit-umfpack` installation on macOS (ARM)

### ISSUE THREAD

**Issue by michaelweinold** [brightway-lca/brightway2-calc#112]
Following [the installation instructions](https://docs.brightway.dev/en/latest/content/installation/index.html#installing-brightway-using-conda-or-mamba)

```bash
conda create -n bw25arm -c conda-forge -c cmutel brightway25_nosolver scikit-umfpack ipykernel
```

and running 

```python
import bw2calc as bc
```

returns

```
[/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/bw2calc/__init__.py:45](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/bw2calc/__init__.py:45): UserWarning: 
It seems like you have an ARM architecture, but haven't installed scikit-umfpack:

    https://pypi.org/project/scikit-umfpack/

Installing it could give you much faster calculations.

  warnings.warn(UMFPACK_WARNING)
```

The version of `bw2calc` installed by the `brightway25_nosolver` is `2.0.DEV23`.

**Comment by michaelweinold** [comment]
...related to a NumPy issue. [The test import in `bw2calc`](https://github.com/brightway-lca/brightway2-calc/blob/0876f0f0cf547a122c2bbd118439538d31e30483/bw2calc/__init__.py#L46C13-L46C35):

```python
import scikits.umfpack
```

returns

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[7], [line 1](vscode-notebook-cell:?execution_count=7&line=1)
----> [1](vscode-notebook-cell:?execution_count=7&line=1) import scikits.umfpack

File /opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/scikits/umfpack/__init__.py:25
     [22](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/scikits/umfpack/__init__.py:22)     del _umfpack_doc, _interface_doc
     [24](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/scikits/umfpack/__init__.py:24) __all__ = [s for s in dir() if not s.startswith('_')]
---> [25](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/scikits/umfpack/__init__.py:25) from numpy.testing import Tester
     [26](https://file+.vscode-resource.vscode-cdn.net/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/scikits/umfpack/__init__.py:26) test = Tester().test

ImportError: cannot import name 'Tester' from 'numpy.testing' (/opt/homebrew/Caskroom/miniconda/base/envs/bw25arm/lib/python3.11/site-packages/numpy/testing/__init__.py)
```

[Which can be resolved by pinning NumPy to `<1.24`](https://stackoverflow.com/a/77563955/).

> [!NOTE]  
> The current workaround is to use:
> 
> ```bash
> conda create -n bw25arm -c conda-forge -c cmutel numpy=1.24.4 brightway25_nosolver scikit-umfpack
> ```

**Comment by cmutel** [comment]
Fixed in https://github.com/brightway-lca/brightway-documentation/commit/417c8b6bb02af5e231d9fb7c9612acfb073aa182
